### DISCUSSION TITLE
Help with flashing

### DISCUSSION THREAD

**Discussion by istalker666** [darthcloud/BlueRetro#680] [General]
Hi, I have the aliexpress blueretro adapter and it comes with gamecube firmware flashed, I was trying to flash the universal firmware to use with all my consoles easily, but sometimes I have to press the reset button and it resets to the factory one (gamecube). is there a way to flash the universal one as factory? Sorry for bad English.

**Comment by darthcloud** [comment]
Flash via USB

https://github.com/darthcloud/BlueRetro/wiki/Flashing-firmware-Windows-10

**Reply by istalker666** [reply]
So I select the WORKMODE factory, right?

**Reply by darthcloud** [reply]
There is screenshots in the wiki page, do the same thing.
